scripts/docker-build

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+# vim: ft=bash
+# Build EVM RPC inside docker. This outputs evm_rpc.wasm.gz.
+
+set -euo pipefail
+
+# Make sure we always run from the root
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$SCRIPTS_DIR/.."
+
+function title() {
+    echo "Build EVM RPC canister inside Docker"
+}
+
+function usage() {
+    cat << EOF
+
+Usage:
+  $0
+
+EOF
+}
+
+function help() {
+    cat << EOF
+
+This will create (and override) "./evm_rpc.wasm.gz".
+EOF
+}
+
+## Building
+
+function build() {
+    local canister="$1"
+
+    # image name and build args, made global because they're used in
+    # check_feature()
+    image_name="evm-rpc-docker-build"
+    docker_build_args=( --target "scratch_$canister" )
+
+    docker_build_args+=(--tag "$image_name" .)
+
+    echo "The following image name will be used: $image_name"
+
+    tmp_outdir=$(mktemp -d)
+
+    local version="$(./scripts/version)"
```

Comment:
Shouldn't this be removed since `./scripts/version` was removed? It's currently not clear to me why the build actually worked 🤔

Reply:
You're right; that is funny that the build still worked. Updated.